qt/scientific_interfaces/Inelastic/Analysis/FunctionBrowser/SingleFunctionTemplateModel.h

```diff
@@ -33,25 +33,25 @@ class MANTIDQT_INELASTIC_DLL SingleFunctionTemplateModel : public FunctionModel
   void setFunction(IFunction_sptr fun) override;
   void updateAvailableFunctions(const std::map<std::string, std::string> &functionInitialisationStrings);
 
-  void setFitType(const QString &name);
-  QString getFitType();
+  void setFitType(const std::string &name);
+  std::string getFitType();
   void removeFitType();
-  QStringList getFunctionList();
+  std::vector<std::string> getFunctionList();
   int getEnumIndex();
-  void setGlobal(const QString &name, bool isGlobal);
+  void setGlobal(std::string const &parameterName, bool isGlobal);
 
   EstimationDataSelector getEstimationDataSelector() const;
   void updateParameterEstimationData(DataForParameterEstimationCollection &&data);
 
   void estimateFunctionParameters();
 
 private:
-  QString m_fitType;
+  std::string m_fitType;
   DataForParameterEstimationCollection m_estimationData;
-  QMap<QString, IFunction_sptr> m_fitTypeToFunctionStore;
-  QMap<QString, QStringList> m_globalParameterStore;
-  QStringList m_fitTypeList;
-  boost::optional<QString> findFitTypeForFunctionName(const QString &name) const;
+  QMap<std::string, IFunction_sptr> m_fitTypeToFunctionStore;
+  QMap<std::string, std::vector<std::string>> m_globalParameterStore;
```

Comment:
some models still retain the QMap type, is there any advantage with respect to std::map, or it is something to perhaps do in a different pr?

Reply:
I think ideally these should be `std::map` as well. In an attempt to keep this PR short (...I tried!) I decided to skip them. Are you happy if we change them in a future PR?